tests/e2e.sh

```diff
@@ -145,7 +145,13 @@ assert_dir_exists knative-serving
 # kpt fn Tests
 ############################
 
-testcase "kpt_set_namespace_success"
+testcase "kpt_set_namespace_go_success"
+kpt pkg get $SDK_REPO/example-configs example-configs
+kpt pkg get $CATALOG_REPO/functions/go ./
+kpt fn run . --enable-exec --exec-path go/set_namespace/set_namespace -- namespace=example-ns
```

Comment:
Can you add another e2e test which does `kpt fn run . --image gcr.io/kpt-functions/set-namespace:"${TAG}" -- namespace=example-ns` so we can test the docker image too?

Reply:
Done